src/pykeen/datasets/cli.py

```diff
@@ -36,23 +41,28 @@ def summarize():
             click.secho(str(e), fg="red", bold=True)
 
 
-def _iter_datasets(regex_name_filter=None):
+def _iter_datasets(regex_name_filter=None, max_triples: Optional[int] = None) -> Iterable[Tuple[str, Type[Dataset]]]:
+    def _get_num_triples(pair: Tuple[str, Type[Dataset]]) -> int:
+        return docdata.get_docdata(pair[1])["statistics"]["triples"]
+
     it = sorted(
         dataset_resolver.lookup_dict.items(),
-        key=lambda pair: docdata.get_docdata(pair[1])["statistics"]["triples"],
+        key=_get_num_triples,
     )
+    if max_triples is not None:
```

Comment:
just because you're a phd now it doesn't mean you can name all of the variables after yourself :p